WordPress/Classes/ViewRelated/Stats/StatsViewController.swift

```diff
@@ -0,0 +1,18 @@
+extension StatsViewController {
+
+    @objc func showJetpackConnectionView(completion: @escaping () -> Void) {
```

Comment:
It needs to be `public` for `Keystone` to compile.

```suggestion
    @objc public func showJetpackConnectionView(completion: @escaping () -> Void) {
```